portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,43 +38,58 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<Comment> commentsList;
-
-  @Override
-  public void init() {
-    commentsList = new ArrayList<>();
-    commentsList.add(Comment.create("Alice", "Wow, this website is great!"));
-    commentsList.add(Comment.create("Bob", "Cool website!"));
-  }
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("comment").addSort("timestamp", SortDirection.DESCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<Comment> comments = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      String name = (String) entity.getProperty("name");
```

Comment:
Define property names as constant vars and put them at the top.

Reply:
Done